smartenvironment/patterns/.htaccess

```diff
@@ -0,0 +1,13 @@
+Options +FollowSymLinks
+RewriteEngine on
+# Link to main Ecare@home patterns page and individual patterns
+RewriteRule ^$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/ [R=302,L]
+RewriteRule ^event.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/event.owl [R=302,L]
+RewriteRule ^geometry.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/geometry.owl [R=302,L]
+RewriteRule ^network.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/network.owl [R=302,L]
+RewriteRule ^object.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/object.owl [R=302,L]
+RewriteRule ^place.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/place.owl [R=302,L]
+RewriteRule ^sensing.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/sensing.owl [R=302,L]
+RewriteRule ^situation.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/situation.owl [R=302,L]
+RewriteRule ^time.owl$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/time.owl [R=302,L]
```

Comment:
Might be easier to use a wildcard here to cover current and future "pattern" names:
```
RewriteRule ^(.*)$ http://ecareathome-ontology.mpi.aass.oru.se/patterns/$1 [R=302,L]
```